### DISCUSSION TITLE
Few components are missing styles

### DISCUSSION THREAD

**Discussion by mittalyashu** [vercel/next.js#18665] [Help]
After reloading the page or at random times.

The styles of the few components doesn't apply.

<img width="100" alt="CleanShot 2020-11-02 at 17 54 17@2x" src="https://user-images.githubusercontent.com/29014463/97867892-b52c0d00-1d34-11eb-9e62-b88e52e4eafd.png">

After checking inside DevTools, the CSS classname does show in the HTML but its styles are missing.

I am using styled-components to apply styles.

**Comment by neg4n** [comment]
Hey, are u doing server side rendering?  
If so, your issue might be fact, that SSR does not fetch styles before re-render of your page. Try configuring your custom `_document` and adding styled-components to `.babelrc` like here: https://github.com/vercel/next.js/tree/canary/examples/with-styled-components

**Reply by mittalyashu** [reply]
Do we still need to enable `ssr: true` in `.babelrc`, even if we're using client-side rendering? @neg4n

**Comment by mittalyashu** [comment]
I also got this error in browser console.

<img width="1291" alt="CleanShot 2020-11-03 at 09 12 18@2x" src="https://user-images.githubusercontent.com/29014463/97947882-bdc42800-1db4-11eb-8a21-71f568b69a94.png">

**Reply by neg4n** [reply]
Check out this issue: https://github.com/vercel/next.js/issues/7423#issuecomment-536779745

**Reply by neg4n** [reply]
@mittalyashu Generally, its hard to guess whats the problem when you provide that small amount of information (ex. no next nor styled-components version was provided). It can be anything (probably it is related to your babel configuration), even https://github.com/vercel/next.js/issues/11600.

Best regards.

**Reply by mittalyashu** [reply]
```json
    "next": "10.0.0",
    "styled-components": "^5.2.0"
```
